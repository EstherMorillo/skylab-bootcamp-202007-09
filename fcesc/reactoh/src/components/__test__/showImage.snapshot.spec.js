import React from 'react';
import renderer from 'react-test-renderer';
import ShowImage from './../ShowImage';

const tree = renderer.create(<ShowImage />).toJSON();

describe('ShowImage snapshot', () => {
    test('should match', () => {
        expect(tree).toMatchSnapshot();
    })
})
