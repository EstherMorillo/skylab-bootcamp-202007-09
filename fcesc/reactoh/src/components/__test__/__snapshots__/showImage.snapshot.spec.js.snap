// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowImage snapshot should match 1`] = `
<img
  alt=""
/>
`;
